hacks/cutNewVersions.sh

```diff
@@ -2,10 +2,10 @@
 set -e
 
 # Before running this script make sure these versions are correct!
-NEW_DOCS_VERSION="8.2"
-PREVIOUS_DOCS_VERSION="8.1"
-NEW_OPTIMIZE_VERSION="3.10.0"
-PREVIOUS_OPTIMIZE_VERSION="3.9.0"
+NEW_DOCS_VERSION="8.3"
+PREVIOUS_DOCS_VERSION="8.2"
+NEW_OPTIMIZE_VERSION="3.11.0"
+PREVIOUS_OPTIMIZE_VERSION="3.10.0"
```

Comment:
These versions are only used for the tooling that cuts a version, but it's nice to commit them so that it's easier to figure out how it works at time of next release.